test/e2e/operator/deployment_api.go

```diff
@@ -464,6 +464,40 @@ var _ = DescribeForAll("pmem-csi", func(o *Operator, f *framework.Framework) {
 			defer deleteDeployment(f, deployment.Name)
 			validateDriverDeployment(f, o, deployment)
 		})
+
+		It("driver deployment shall be running even after operator exit", func() {
+			dep := &unstructured.Unstructured{
+				Object: map[string]interface{}{
+					"apiVersion": api.SchemeGroupVersion.String(),
+					"kind":       "Deployment",
+					"metadata": map[string]interface{}{
+						"name": "test-deployment-operator-exit",
+					},
+					"spec": map[string]interface{}{
+						"driverName": "operator-exit.com",
+						"image":      dummyImage,
+					},
+				},
+			}
+
+			deployment, err := toDeployment(dep)
+			Expect(err).ShouldNot(HaveOccurred(), "unstructured to deployment conversion")
+
+			createDeployment(f, dep)
+
+			defer deleteDeployment(f, deployment.Name)
+			validateDriverDeployment(f, o, deployment)
+
+			c, err := deploy.NewCluster(f.ClientSet)
+			Expect(err).ShouldNot(HaveOccurred(), "create new cluster")
+
+			By("Deleting operator")
+			// Stop the operator
+			EnsureOperatorRemoved(c, o)
```

Comment:
Did the changes to delete only operator Deployment object without deleting the roles/role bindings. This avoids the case of the CRD exists just because of lack of permissions to delete.